### PR TITLE
wip(test): Remove convey

### PR DESCRIPTION
**Pull request by gfanton**
replace convey entirely with [assert](https://github.com/stretchr/testify)

### CODE REVIEW

File: tests/create_open_test.go [hunk 1]
```diff
@@ -33,143 +34,143 @@ func TestCreateOpen(t *testing.T) {
...
 	ipfs := testingCoreAPI(t, node)
 
-	Convey("orbit-db - Create & Open", t, FailureHalts, func(c C) {
+	t.Run("orbit-db - Create & Open", func(t *testing.T) {
 		dbPath, clean := testingTempDir(t, "db")
 		defer clean()
 
 		orbit, err := orbitdb.NewOrbitDB(ctx, ipfs, &orbitdb.NewOrbitDBOptions{Directory: &dbPath})
-		c.So(err, ShouldBeNil)
+		assert.NoError(t, err)
```
Comment (moul) [inline tests/create_open_test.go]: If i remember correctly, asset.NoError won't stop, you should make something like

```golang
If !assert.NoError(t, err) {
    return
}
```

Comment (moul) [reply tests/create_open_test.go]: I made an helper in berty/berty, look for `func checkErr(t *testing.T, err error)`

**Comment by moul** [comment]
@gfanton this PR is now conflicted, what do you prefer, close + redo later?

**Comment by gfanton** [comment]
yes, it's preferable to close this one and redo it later

**Pull request closed without merging**